rrfs-test/validated_yamls/templates/obtype_config/adpupa_airTemperature_120.yaml

```diff
@@ -0,0 +1,221 @@
+     - obs space:
+         name: adpupa_airTemperature_120
+         distribution:
+           name: "@DISTRIBUTION@"
+           halo size: 100e3
+         obsdatain:
+           engine:
+             type: H5File
+             obsfile: "@OBSFILE@"
+           obsgrouping:
+             group variables: ["stationIdentification"]
+             sort variable: "pressure"
+             sort order: "descending"
+         obsdataout:
+           engine:
+             type: H5File
+             obsfile: hofx_adpupa_airTemperature_120.nc4
```

Comment:
Should we use the new `jdiag` format that we switched to in #241?

Reply:
I wasn't sure if now or in a later commit where we can change them all would be better. I just went ahead and did it now.